lib/proxy/archive_extract.rb

```diff
@@ -0,0 +1,30 @@
+module Proxy
+  class ArchiveExtract < Proxy::Util::CommandTask
+    include Util
+
+    def initialize(image_path, file_in_image, dst_path)
+      args = [which('isoinfo')]
+
+      # read the file
+      args << "-R"
```

Comment:
Is this correct?
```console
$ isoinfo --help 2>&1 | grep -- -R
	-R		Print information from Rock Ridge extensions
	dev=target	SCSI target to use as CD/DVD-Recorder
```

Reply:
You're right, I applied this from a tutorial and I think we don't need it here.